apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJobIT.java

```diff
@@ -28,14 +30,23 @@
 public final class CcwRifLoadJobIT {
   private static final Logger LOGGER = LoggerFactory.getLogger(CcwRifLoadJobIT.class);
 
+  /** only need a single instance of the S3 client. */
+  private static AmazonS3 s3Client;
+
+  /** Sets the minio test container. */
+  @BeforeAll
+  public static void setupMinioTestContainer() {
+    MinioTestContainer.getInstance();
+    s3Client = S3Utilities.createS3Client(new ExtractionOptions("foo"));
```

Comment:
I guess you need to put something in the Extraction options, that is why foo is there?

Reply:
Actually, almost every call to createS3Client passed in that arg (foo); basically most of the changes here were simply removing 'n' calls to create the client and replacing with a single one-time creation for an entire IT class. Not entirely sure why all the createS3Client used the ExtraOptions("foo").

Reply:
gotcha

Reply:
Actually, that code is no longer even in the CcwRifLoadJobIt.java source. So nothing to change there since that IT is derived from the MinioTestContainer.